### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by sjpadgett**
Fixes #

### CODE REVIEW

**Comment by sjpadgett** [comment]
@bradymiller

File: portal/patient/scripts/app/onsitedocuments.js [hunk 1]
```diff
@@ -951,5 +955,16 @@ var page = {
                 app.hideProgress('modelLoader');
             }
         });
+    },
+    encode: function (content, k) {
+        let encoded = "";
+        content = btoa(content);
+        content = btoa(content);
```
Comment (stephenwaite) [inline portal/patient/scripts/app/onsitedocuments.js:962]: hi @sjpadgett , this looks redundant but could be wrong :)

File: portal/patient/libs/Controller/OnsiteDocumentController.php [hunk 2]
```diff
@@ -379,4 +384,21 @@ public function Delete()
...
+    private function decode($encoded, $v): bool|string
+    {
+        $encoded = base64_decode($encoded);
+        $decoded = "";
+        for ($i = 0; $i < strlen($encoded); $i++) {
+            $b = ord($encoded[$i]);
+            $a = $b ^ $v;
+            $decoded .= chr($a);
+        }
+        return base64_decode(base64_decode($decoded));
```
Comment (stephenwaite) [inline portal/patient/libs/Controller/OnsiteDocumentController.php:402]: hi @sjpadgett , why would you `base64_decode` twice?

Comment (sjpadgett) [reply portal/patient/libs/Controller/OnsiteDocumentController.php:402]: Check out the bitwise.

**Comment by sjpadgett** [comment]
need for demos and client testing.

**Pull request merged**